components/ui/NavItem.jsx

```diff
@@ -0,0 +1,18 @@
+import Link from "next/link";
+import classNames from "classnames";
+import styles from "./Navbar.module.css";
+
+
+const NavItem=({title,link,isActive,index})=>{
+
+    return(
+        <li
+            className={classNames(styles.navbarLinks, {
+              [styles.isLinkActive]:isActive})}>
+        <Link href={link} title={title} >
+        <b>0{index + 1} </b>{title}
```

Comment:
This will stop working if the index is above 9 😄

Reply:
Yes, agreed your point. I did not think in the large scope. Now i have modified my code. Thanks for pointing it out.